pages/linux/rofi.md

```diff
@@ -13,3 +14,7 @@
 - Switch between windows:
 
 `rofi -show window`
+
+- If we pipe a list of items to stdin, the selected item will be printed (or piped into a different process) to stdout:
```

Comment:
Sorry to be a pain, but I'm not so sure about this. I think it's the `If` here that's bothering me. What about this?

```suggestion
- Pipe a list of items to stdin and print the selected item stdout:
```

I think we can assume that the reader will know that they can send the standard output of one command into another as the standard input.

Reply:
It is not clear to me how the item is selected. The stdin input is just a list of lines.

Reply:
Rofi will open a ui thing that'll let the user select one

Reply:
Ok, then we should clarify that. Perhaps say "select an item and then print it to console" or something to that effect.